default/python/AI/AIDependencies.py

```diff
@@ -77,21 +77,21 @@
 # boni that scale with planet size
 POP_SIZE_MOD_MAP = {
     # "Effect": [uninhab, hostile, poor, adequate, good]
-    "environment_bonus": [0,-4,-2, 0, 3],
-    "GRO_SUBTER_HAB":    [0, 1, 1, 1, 1],
+    "environment_bonus": [0, -4, -2, 0, 3],
+    "GRO_SUBTER_HAB": [0, 1, 1, 1, 1],
     "GRO_SYMBIOTIC_BIO": [0, 0, 1, 1, 1],
     "GRO_XENO_GENETICS": [0, 1, 2, 2, 0],
-    "GRO_XENO_HYBRID":   [0, 2, 1, 0, 0],
-    "GRO_CYBORG":        [0, 2, 0, 0, 0],
-    "CON_NDIM_STRUC":    [0, 2, 2, 2, 2],
-    "CON_ORBITAL_HAB":   [0, 1, 1, 1, 1],
+    "GRO_XENO_HYBRID": [0, 2, 1, 0, 0],
+    "GRO_CYBORG": [0, 2, 0, 0, 0],
+    "CON_NDIM_STRUC": [0, 2, 2, 2, 2],
+    "CON_ORBITAL_HAB": [0, 1, 1, 1, 1],
```

Comment:
I myself do have a tendency to align things (and probably created the aligned table in question above), and in cases where there is some notable need to compare within columns across rows, then I might argue in support of such alignment.  In this kind of case, though, I have become persuaded that to the extent the previous version seems 'clearer' or 'cleaner', it is really a more abstract 'prettiness' being contemplated and not actual readability of the given lines. 

This kind of issue is a common point of contention, and even within the pycodestyle community there seems to be a lot of [argument](https://github.com/PyCQA/pycodestyle/issues/381) about how much support should be provided for selectively disabling individual pep8 rules on a line-by-line or file-by-file basis.  

So, to the extent our tools support it, we could consider disabling certain checks for certain parts of code (but should really really minimize any such practice).   For the most part, though, I see a lot of benefit to following the standard in a straightforward fashion.  And so far, in the instances pointed to here, I am not convinced of any significant value to not following PEP8.